app/frontend/src/features/auth/authActions.ts

```diff
@@ -43,3 +43,12 @@ export const signup = createAsyncThunk(
     return { token: auth.token, jailed: auth.jailed, user: null };
   }
 );
+
+export const tokenLogout = createAsyncThunk(
```

Comment:
```suggestion
export const logout = createAsyncThunk(
```
I think you can name this just `logout` since there's no other means to authenticate against the API.

Reply:
Got it. I didn't understand how to correctly use extraReducers to combine logic. Should be correct now.